### DISCUSSION TITLE
How to study this book??

### DISCUSSION THREAD

**Discussion by franegrao** [giswqs/geebook#7] [Q&A] [answered]
Hi,
I am having a lot of trouble in retaining the information conveyed in the book. I am only in chapter 3 and already feeling lost. I think I'd need something like exercises to make things more engaging and memorable. I feel I am not getting very far by tweaking output and taking notes. Any ideas?

**Comment by giswqs** [comment, marked as answer]
What specific troubles are you having? 

If you want more exercises, you can check out https://geog-414.gishub.org/book/labs/lab_02.html

For video tutorials, check out https://www.youtube.com/playlist?list=PLAxJ4-o7ZoPeeJZL-_3kcZ2WLYF5x87eX

**Reply by ghost** [reply]
Hi there, I'm quite new to coding. I'm dealing with a similar coding challenge. Can plug-and-play coding be effectively implemented with the geemap library with close reference to resources I discovered as below: 
-	Ebook (https://locatepress.com/book/geemap) ,
-	youtube video channel (https://www.youtube.com/channel/UC6V9oXSB3T1XSBsbqtYvKMg) , 
-	website (https://geemap.org/) 
Also, is it a recommended practice? Is there any other important resources that I missed out? Any additional advice or guidance would be greatly appreciated.

Through the process of exploration, it came to my attention that while Mamba may be easy for beginners to use; it does not exhibit the same level of stability as Miniconda or Anaconda. 

P/S: Let me know if anyone find the content of this post is displeasing.

**Reply by ghost** [reply]
Hi, i found other reference sources about GEE and geemap [workshop](https://www.youtube.com/watch?v=oHVlWqkun-Y). 
thanks =)

**Comment by franegrao** [comment]
Thank you very much for the links, especially the one to the exercises.
I kind of get lost in a sea of examples and at some point and lose the big picture (I have been studying the material intermittently and maybe partly because of that, I miss a "general map of the knowledge", which of course underlies the exposition. I'll make one, (which is not difficult and a good way to learn) so I understand where I am next time, without having to read go through the examples/explanations one by one.
Thank you for helping out!
